file_downloader.py

```diff
@@ -0,0 +1,51 @@
+import getopt
+import glob
+import os.path
+import sys
+import urllib.request
+import zipfile
+
+def get_urls(inputfile):
+    file = open(inputfile, "r")
+    urls = file.read().splitlines()
+    return urls
+
+def download_files(url):
+    local_filename = url.split('/')[-1]
+
+    if not os.path.exists(local_filename):
+      print("Downloading file {0}\n".format(local_filename))
+      urllib.request.urlretrieve(url, local_filename)
+
+      if zipfile.is_zipfile(local_filename):
```

Comment:
If the local file exists, will this overwrite?

Reply:
Shouldn't get that far